tests/unit/test_stores.py

```diff
@@ -166,7 +169,7 @@ async def test_expires_in(store: Store, frozen_datetime: FrozenDateTimeFactory)
     assert expiration is not None
     assert math.ceil(expiration / 10) == 1
 
-    frozen_datetime.tick(12)
+    await store._redis.expire(f"{store.namespace}:foo", 0)
```

Comment:
This needs a check to see if it's actually a redis store

Reply:
I did have a check there, but the condition for xfail here https://github.com/litestar-org/litestar/pull/2030/files#diff-8d1aade4d9d5b80560a1e0d88ee2bf52ad65ec295584ac7a95c80694536e278eR138 made mypy complain about unreachable code

Reply:
Oh, I missed that!